package.json

```
@@ -5,7 +5,7 @@
     "type": "module",
     "scripts": {
         "build": "next build",
-        "db:push": "prisma migrate dev",
+        "db:push": "prisma db push",
```

Small mistake when merging?

```suggestion
        "db:push": "prisma migrate dev",
```

---

😇